src/kses.php

```diff
@@ -8,6 +8,29 @@
 	exit;
 }
 
+/**
+ * In WooCommerce Shop page, <style> tags are stripped out and the CSS styles are displayed in the frontend.
+ * This function removes the <style> tags and CSS styles before they are stripped out.
+ */
+if ( ! function_exists( 'stackable_pre_kses_woocomerce_shop' ) ) {
```

Comment:
Can we move this code from here to the new `src/compatibility/woocommerce.php` file suggested above?